modular_sojourn/area_blurbs.dm

```diff
@@ -0,0 +1,157 @@
+
+//Maintenance
+
+/area/nadezhda/maintenance
+	narrate = "Dark passages long abandoned by the colony were left to be the domain of roaches, rats and spiders..."
+
+/area/nadezhda/maintenance/junk
+	narrate = "The Junkyard, maintained by the local faith. Watch out for intergalactical scrap meteors being forcibly pulled by its force."
+
+/area/nadezhda/maintenance/arcade
+	narrate = "Once a place of fun and games, now rotted and rusted."
+
+//Outside abandoned structures
+
+/area/nadezhda/dungeon/outside/trashcave
+	narrate = "Roaches and critters have made this damp and long abandoned cave their nest. A glint of gold shimmers from within the piles of scrap and rock walls, awaiting the curious prospector to unearth its hidden treasures."
+
+/area/nadezhda/dungeon/outside/abandoned_solars
+	narrate = "Its position being a difficult one to defend and properly mantain, these solar panels have long since fallen into abandonement, ravaged by local fauna and the unforgiving passage of time. A couple lengths of wire and a passion for construction will mayhaps render this decaying piece of civilization functional once more..."
+
+/area/nadezhda/dungeon/outside/burned_outpost
+	narrate = "Perhaps a mere accident, or an intential arson to cover deeds of dubious nature, this small outpost has charred to the ground now, leaving behind only the faintest traces of whatever life inhabited it before."
+
+/area/nadezhda/dungeon/outside/hunter_cabin
+	narrate = "The smell of rugged wood and butchered game fill the air. The ambience itself speaks of survival, of hearth and safety. Of home. A local hunter might have previously made this place their abode, albeit seemingly not for long..."
+
+/area/nadezhda/dungeon/outside/farm
+	narrate = "A humble plot of land ruined by local fauna and left to the mercy of local weeds and abandonement of its soil. With labor and dedication enough, you might make these lands a profitable farm once more..."
+
+/area/nadezhda/dungeon/outside/abandoned_outpost
+	narrate = "The earliest traces of sentient technological settlement on the planet can be found in what's left of this haphazardly built outpost, now fallen to ruin. Be wary of the roaches and spiders that have claimed it as their nest."
+
+/area/nadezhda/dungeon/outside/zoo
+	narrate = "Sounds of different fauna, feral growls and cute squeaks of all kinds, can be heard echoing through the otherwise silent emptiness of this abandoned facility. Their purpose unknown, its mantainers nowhere to be seen, it now stands as the ruins of an animal-holding facility colloquially dubbed as 'The zoo'."
+
+/area/nadezhda/dungeon/outside/prepper
+	narrate = "A base of operations fallen into disrepair, its former inhabitants hoarding all manner of gear for survival have long since left it on the care of their defensive security systems, long since gone haywire. From its insides, a chittering that's definitely not robotic in nature can faintly be heard..."
```

Comment:
```suggestion
	narrate = "A base of operations fallen into disrepair, its former inhabitants hoarding all manner of gear for survival have long since left it on the care of their defensive security systems, now gone haywire. From its insides, a chittering that's definitely not robotic in nature can faintly be heard..."
```
I didn't notice I just typed that twice, my bad